example/contract-originate-template.ts

```diff
@@ -0,0 +1,57 @@
+import { TezosToolkit } from '@taquito/taquito';
```

Comment:
Can we add a comment about where this contract is used?

Reply:
added "//This contract is used in integration tests where origination of contracts using voting is tested."